### PR TITLE
demo/tooltip-extra-information

### PR DESCRIPTION
**Pull request by karolkolodziej**
Added API demo showing how to display extra information in the tooltip.

### CODE REVIEW

File: samples/highcharts/tooltip/formatter-extra-information/demo.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    xAxis: {
+        categories: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+    },
+
+    series: [{
+        data: [
+            29.9,
+            {
+                y: 71.5,
+                extraInfrmation: 'First month when XYZ introduced.'
```
Comment (raf18seb) [inline samples/highcharts/tooltip/formatter-extra-information/demo.js:22]: I think we should push the users to add additional info in a `data.custom` that was added specifically for cases like this
https://api.highcharts.com/highcharts/series.line.data.custom

File: samples/highcharts/tooltip/formatter-extra-information/demo.details [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+ name: Highcharts Demo
+ authors:
+   - Torstein Hønsi
```
Comment (raf18seb) [inline samples/highcharts/tooltip/formatter-extra-information/demo.details:4]: Don't you want to change the author?

**Review by raf18seb — CHANGES_REQUESTED** [review]
Please see comments below

File: samples/highcharts/tooltip/formatter-extra-information/demo.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+
+<div id="container" style="height: 400px"></div>
```
Comment (TorsteinHonsi) [inline samples/highcharts/tooltip/formatter-extra-information/demo.html:3]: Avoid style attributes, use a `demo.css` file instead.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks for this useful demo!

I think we should encourage our users to use `pointFormat` instead of `formatter`. That's because it is JSON-compatible and thus more secure, avoiding callbacks in the options structure, and more compliant to wrappers and other languages. 

Arguably, formatters are more flexible, but in this case the implementer has control over `extraInformation`, so any logic can be added where this property is generated.

This is the equivalent setup with `pointFormat` instead of `formatter`: https://jsfiddle.net/highcharts/68pmut32/. As a consequence, the name of the sample must change too.

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**